docs/Release to Google Play.md

```diff
@@ -0,0 +1,57 @@
+# Release to Google Play
+
+## Manual Build
+
+These instructions follow steps from the [React Native docs](https://reactnative.dev/docs/signed-apk-android.html), but adds in things specific to the CovidShield codebase.
+
+### What you'll need
+
+- Keystore file
+- Store password
+- Key password
+- Key alias
+
+### Place the keystore file
+
+Copy your keystore file into `android/app`. Take note of the full path to where you copied it to, including the file name.
+
+### Configure production settings
+
+First, you'll need to create a `.env.production` file at the base of your repository. You can copy the `.env` file in the base of the repository to get started. Add in the appropriate information, and uncomment the block around creating an Android release.
+
+```
+# For Release Builds to Google Play
+# PRODUCT_BUNDLE_IDENTIFIER=<product bundle identifier>
+# COVIDSHIELD_UPLOAD_STORE_FILE=<path to keystore file>
+# COVIDSHIELD_UPLOAD_KEY_ALIAS=<key alias>
+# COVIDSHIELD_UPLOAD_STORE_PASSWORD=<store password>
+# COVIDSHIELD_UPLOAD_KEY_PASSWORD=<keystore password>
+```
+
+For the `APP_ID_ANDROID` value, make sure this matches your app in the Google Play Console. Additionally, you'll need to make sure `APP_VERSION_CODE` is at least `1` higher than what you've published in the past.
+
+### Build the Android Application Bundle
+
+Run the following commands:
+
+1. Clean up past builds: `yarn build-android-clean`
+1. Compile and sign a release: `yarn build-android-release`
+
+### Get the built bundle
+
+Assuming the build and compilation went well, you should now have an `AAB` file in this location:
+
+`android/app/build/outputs/bundle/release/app-release.aab`
+
+### Test the build
+
+You can launch a release version of your application with the following command:
+
+```
+yarn run-android-release
```

Comment:
While this installs the app for me, the app immediately closes after opening. Does this work for y'all?

Reply:
This is the error message I get:

```
BUILD SUCCESSFUL in 1m 49s
329 actionable tasks: 5 executed, 324 up-to-date
info Connecting to the development server...
info Starting the app on "emulator-5554"...
Starting: Intent { cmp=app.covidshield/.MainActivity }
Error type 3
Error: Activity class {app.covidshield/app.covidshield.MainActivity} does not exist.
✨  Done in 116.07s.
```

To double check, I've tried the build steps both via command line and Android Studio, but get the same error either way.

Reply:
Since we set the APP_ID_ANDROID to something else (i.e. not `app.covidshield`, maybe it's looking to start the wrong activity (i.e. this has the app.covidshield stuff from the CovidShield repository). Trying to find something.

Reply:
I get the same error message on the debug run, but the application starts up fine, so this error likely isn't important. It must be something else on the release build.

Reply:
Tried a few other things, pushed updates. Issue persists.